### PR TITLE
added docs for React and vue2/3

### PR DESCRIPTION
**Pull request by vinayaksh42**

### CODE REVIEW

**Comment by kbhutani0001** [comment]
@vinayaksh42 can you attach some screenshot or video for the same?

**Comment by vinayaksh42** [comment]
[Get EOS Icons _ EOS Icons vue.pdf](https://github.com/EOS-uiux-Solutions/eos-icons-landing/files/7015042/Get.EOS.Icons._.EOS.Icons.vue.pdf)
[Get EOS Icons _ EOS Icons.pdf](https://github.com/EOS-uiux-Solutions/eos-icons-landing/files/7015044/Get.EOS.Icons._.EOS.Icons.pdf)
@kbhutani0001

File: .env [hunk 1]
```diff
@@ -1 +1 @@
-PUBLIC_URL=https://test.eos-icons.com
+PUBLIC_URL=https://test.eos-icons.com
```
Comment (cyntss) [inline .env:1]: can you remove the "test" from this URL? I know this isn't a change coming from you but it is good that we saw it now!

File: src/pages/Docs.js [hunk 2]
```diff
@@ -449,6 +449,510 @@ const Docs = () => {
               />
             </div>
           </div>
+          <div label='React'>
+            <div className='container'>
+              <h2>Installing EOS icons React</h2>
+              <p>
+                Note: upgrade to eos-icons-react@2.1.1 as the version@2.1.0 has
```
Comment (cyntss) [inline src/pages/Docs.js:456]: I would rephrase this to be more positive:
Note: we've introduced several upgrades in eos-icons-react@2.1.1 and we strongly suggest upgrading.

File: src/pages/Docs.js [hunk 3]
```diff
@@ -449,6 +449,510 @@ const Docs = () => {
...
+                  rel='noopener noreferrer'
+                >
+                  {' '}
+                  <i className='eos-icons eos-18'>open_in_new</i>
+                </a>
+              </h3>
+              <pre className='code language-shell'>
+                <code>yarn add eos-icons-react</code>
+              </pre>
+              <h2>Using EOS icons React in your projects</h2>
```
Comment (cyntss) [inline src/pages/Docs.js:493]: can we add a <p> here that says

Note: the middle part of the component name is the same as the icon name and should always be written in uppercase.

**Review by cyntss — CHANGES_REQUESTED** [review]

**Review by cyntss — APPROVED** [review]
